elm-format-lib/src/Parse/Literal.hs

```diff
@@ -125,7 +97,7 @@ chr =
 
     toError e = newErrorUnknown ("Error parsing char: " ++ show e)
   in
-  do  s <- character toExpecation toError
+  do  s <- Parse.String.character newErrorUnknown newErrorUnknown
```

Comment:
This is a silly mistake that does not compile, `[...] toExpectation toError` should not have been replaced. Fixed in ef5ef6a